src/components/area-plot/AreaPlot.js

```diff
@@ -1,100 +1,183 @@
-
-  const xAxisDataKey = getXAxisDataKey(plotConfig);
+function PivotedAreaPlot({ plotConfig }) {
+  const data = getData(plotConfig);
   const categoryAxisDataKey = getCategoryAxisDataKey(plotConfig);
   const uniqueValuesOfCategoryKey = getUniqueValuesOfDataKey(plotConfig, categoryAxisDataKey);
+  const showDataAnnotations = getSeriesShowDataAnnotations(plotConfig);
+  const yAxisTickFormatter = getYAxisTickFormatter(plotConfig);
+  const dataAnnotationsChangeType = getAreaPlotDataAnnotationsChangeType(plotConfig);
+  return uniqueValuesOfCategoryKey.map((uniqueValueOfCategoryKey, index) => (
+    <Area
+      stroke={PLOT_COLORS[index % PLOT_COLORS.length]}
+      fill={PLOT_SECONDARY_COLORS[index % PLOT_SECONDARY_COLORS.length]}
+      dataKey={uniqueValueOfCategoryKey}
+      type="monotone"
+      strokeWidth={2}
+      name={uniqueValueOfCategoryKey}
+      key={uniqueValueOfCategoryKey}
+      stackId="1"
+      isAnimationActive={false}
+      label={
+        showDataAnnotations ? (
+          <StackedDataAnnotation
+            dataKey={uniqueValueOfCategoryKey}
+            categoryKeyValues={uniqueValuesOfCategoryKey}
+            data={data}
+            dataChangeType={dataAnnotationsChangeType}
+            valueFormatter={yAxisTickFormatter}
+            getCurrentValue={(dataIndex, dataKey) => data[dataIndex][dataKey]}
+            getTotalValue={(dataIndex) =>
+              uniqueValuesOfCategoryKey.reduce(
+                (total, categoryKeyValue) => data[dataIndex][categoryKeyValue] + total,
+                0
+              )
+            }
+          />
+        ) : undefined
+      }
+    />
+  ));
+}
 
-  const yAxisDataKey = getYAxisDataKey(plotConfig);
-
+function NonPivotedAreaPlot({ plotConfig }) {
+  const categoryValueAxes = getCategoryValueAxes(plotConfig);
+  const showDataAnnotations = getSeriesShowDataAnnotations(plotConfig);
+  const dataAnnotationsChangeType = getAreaPlotDataAnnotationsChangeType(plotConfig);
+  const data = getData(plotConfig);
+  return categoryValueAxes.map((axis, index) => (
+    <Area
+      type="monotone"
+      dataKey={axis.dataKey}
+      name={axis.name}
+      key={axis.name}
+      fill={PLOT_SECONDARY_COLORS[index % PLOT_SECONDARY_COLORS.length]}
+      stroke={PLOT_COLORS[index % PLOT_COLORS.length]}
+      strokeWidth={2}
+      stackId="1"
+      isAnimationActive={false}
+      label={
+        showDataAnnotations ? (
+          <StackedDataAnnotation
+            data={data}
+            dataKey={axis.dataKey}
+            getCurrentValue={(dataIndex, dataKey) => data[dataIndex][dataKey]}
+            getTotalValue={(dataIndex) =>
+              categoryValueAxes.reduce(
+                (total, categoryValueAxis) => data[dataIndex][categoryValueAxis.dataKey] + total,
+                0
+              )
+            }
+            dataChangeType={dataAnnotationsChangeType}
+            valueFormatter={getTickFormatterFromDataKey(plotConfig, axis.dataKey)}
```

Comment:
Something interesting - I originally named this prop `formatter`, but because `recharts` is cloning this component and passing in its own props, and one of those happens to be `formatter`, it was overwriting my function. Tricky to troubleshoot!